pkg/cloudprovider/provider/gce/config.go

```diff
@@ -111,6 +113,8 @@ type config struct {
 	machineType    string
 	diskSize       int64
 	diskType       string
+	network        string
```

Comment:
Where do you retrieve these values from cloud provider spec?

Reply:
ups, I probably missed that

Reply:
fixed